segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/StreamSegmentServiceNoOpWriteReadTests.java

```diff
@@ -65,4 +65,12 @@ protected ServiceBuilder createBuilder(ServiceBuilderConfig.Builder builderConfi
                              .withStorageFactory(setup -> this.storageFactory)
                              .withDataLogFactory(setup -> this.durableDataLogFactory);
     }
+
+    /**
+     * This method intentionally left blank as it's out of concern for No-Op Storage.
```

Comment:
Is it possible to define this method as empty block in the base class itself so that we don't have to keep overriding in derived classes.

Reply:
Done.